### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n, @zendesk/mintegrations 

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -4,17 +4,83 @@ packages:
...
   - translation:
-      key: "txt.apps.magento.app.description"
-      title: "Displays customer and order details from Magento"
-      value: "Displays customer and order details from Magento."
-  - translation:
       key: "txt.apps.magento.app.name"
-      title: "App name"
+      title: "Name for App: Magento"
       value: "Magento"
-      hidden: true
```
Comment (injuhwang) [inline translations/en.yml]: Please reinstate this.

File: translations/en.yml [hunk 2]
```diff
@@ -4,17 +4,83 @@ packages:
...
+  - translation:
+      key: "txt.apps.magento.app.short_description"
+      title: "Description for App: Magento"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Magento.\"

File: translations/en.yml [hunk 3]
```diff
@@ -4,17 +4,83 @@ packages:
...
+      value: "Displays customer and order details from Magento."
+  - translation:
+      key: "txt.apps.magento.app.long_description"
+      title: "Long Description for App: Magento"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Magento.\"

File: translations/en.yml [hunk 4]
```diff
@@ -4,17 +4,83 @@ packages:
...
+
+
+* Easily collaborate to resolve customer issues faster with access to support ticket information in a custom Zendesk Support panel within Magento.
+
+* Stay up-to-date on the account and the customer with a single view of support information next to critical sales statistics, customer profiles, and sales orders.
+
+* Create and update Zendesk Support tickets directly in Magento for greater Support team efficiencies.
+
+
+For complete setup instructions, see the [setup guide](https://support.zendesk.com/entries/22329981-magento-integration-installing-configuring-the-magento-extension).
```
Comment (injuhwang) [inline translations/en.yml]: Please replace the URL with this: https://support.zendesk.com/hc/en-us/articles/203660046
It'll help us to track down in the future when we need to edit it.

File: translations/en.yml [hunk 5]
```diff
@@ -4,17 +4,83 @@ packages:
...
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.magento.app.installation_instructions"
+      title: "Installation Instructions for App: Magento"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Magento.\"

File: translations/en.yml [hunk 6]
```diff
@@ -4,17 +4,83 @@ packages:
...
+
+* Magento Community v1.7+ or Magento Enterprise v1.9+ with full administrator rights
+
+* PHP 5.2
+
+
+### How to install
+
+
+The *Zendesk Support for Magento* setup has two parts.  First, sign in to your Zendesk Support account, then follow the [instructions here](https://support.zendesk.com/entries/22329981-magento-integration-installing-configuring-the-magento-extension) to download the extension and learn how to configure the extension in Magento.  When this is complete, you are ready to add the *Magento App for Zendesk Support* to your account.
```
Comment (injuhwang) [inline translations/en.yml]: Please replace the URL with this: https://support.zendesk.com/hc/en-us/articles/203660046
It'll help us to track down in the future when we need to edit it.

File: translations/en.yml [hunk 7]
```diff
@@ -4,17 +4,83 @@ packages:
...
+* **Store URL**: The URL to your Magneto store (for example, http://www.zendesk-store.com).
+
+* **Store Code**: Add the default store code you use, if you have 'Add Store Code to Urls' set to 'Yes' on your Magento store.
+
+* **Access token**: Your API access token from the Zendesk Support extension for Magento.
+
+* **Order ID Field ID**: The Zendesk Support custom field ID you are using to store the associated Magento order number.
+
+
+For complete instructions, see the [setup guide](https://support.zendesk.com/entries/22347936-magento-integration-installing-the-magento-app-for-zendesk)."
```
Comment (injuhwang) [inline translations/en.yml]: Please replace the URL with this: https://support.zendesk.com/hc/en-us/articles/203660066
It'll help us to track down in the future when we need to edit it.

**Comment by pmgrove** [comment]
@injuhwang 💇

File: translations/en.yml [hunk 8]
```diff
@@ -4,18 +4,85 @@ packages:
...
+  - translation:
+      key: "txt.apps.magento.app.long_description"
+      title: "Long Description for App: Magento. Do not translate \"Magento.\""
+      value: "The Zendesk Support for Magento app works to unite your business with a dynamic two-way integration between Zendesk Support ticket data and Magento e-commerce data.
+
+
+### Improve customer satisfaction with better conversations
+
+
+* Manage all your customer interactions in one place, whether you have one Magento site or several—driving efficiencies within your Support team.
```
Comment (HiroIguchi) [inline translations/en.yml]: "several—driving" to be "several-driving" (using dash)?
Or space needs before and after "—" like "several — driving"?

**Comment by HiroIguchi** [comment]
Please create a package as like https://github.com/zendesk/logmein_app/pull/3

File: translations/en.yml (hunk 8 above)
Comment (pmgrove) [inline translations/en.yml]: I think we can just remove the `-` character

**Comment by pmgrove** [comment]
@HiroIguchi 💇

**Comment by mariasica** [comment]
👍

**Pull request merged**